src/main/java/mate/academy/service/impl/ShoppingCartServiceImpl.java

```diff
@@ -0,0 +1,54 @@
+package mate.academy.service.impl;
+
+import mate.academy.dao.ProductDao;
+import mate.academy.dao.ShoppingCartDao;
+import mate.academy.lib.Inject;
+import mate.academy.lib.Service;
+import mate.academy.model.Product;
+import mate.academy.model.ShoppingCart;
+import mate.academy.service.ShoppingCartService;
+
+@Service
+public class ShoppingCartServiceImpl implements ShoppingCartService {
+    @Inject
+    private ShoppingCartDao shoppingCartDao;
+
+    @Inject
+    private ProductDao productDao;
```

Comment:
we don't use this variable anywhere in this class

Reply:
done)